backend/Squirrel.ConsoleApp/Extensions/HubConnectionExtensions.cs

```diff
@@ -0,0 +1,178 @@
+﻿using Microsoft.AspNetCore.SignalR.Client;
+using Squirrel.ConsoleApp.BL.Interfaces;
+using Squirrel.ConsoleApp.Models;
+using Squirrel.ConsoleApp.Services;
+using System.Threading.Channels;
+
+namespace Squirrel.Core.WebAPI.Extensions;
+
+public static class HubConnectionExtensions
+{
+    private static bool CurrentClientIsTarget(string currentClientId, string clientId)
+    {
+        if (string.IsNullOrEmpty(currentClientId))
+        {
+            return false;
+        }
+
+        if (!clientId.Equals(currentClientId))
+        {
+            return false;
+        }
+
+        return true;
+    }
+
+    public static void RegisterActions(this HubConnection hubConnection, string currentClientId, IGetActionsService getActionsService)
+    {
+        hubConnection.On("GetAllTablesNamesAsync", (string clientId) =>
+        {
+            if (!CurrentClientIsTarget(currentClientId, clientId))
+            {
+                return;
+            }
```

Comment:
this check is present in all lambdas, so extract a method with a handler delegate as a parameter or something like this to avoid check repetition

Reply:
Decided to leave this implementation. It's will be more understandeble and I added one more repetitive string (for creating new service instance for every call). It may be refactored next time at another task/PR.

Reply:
@Zoranais and I have solved problem with notifying all clients. For now we use Client Id to identify the target of notification. In this case we don't need additional checks anymore and I have already deleted them.